.github/workflows/reusable_pull_request.yml

```diff
@@ -191,3 +195,18 @@ jobs:
       uses: actions/checkout@v4
     - name: Run format check
       run: swift format lint --parallel --recursive --strict
+
+  cxx-interoperability-check:
+    name: Cxx interoperability check
+    if: ${{ inputs.cxx_interoperability_check_enabled }}
+    runs-on: ubuntu-latest
+    container:
+      image: swift:5.10-noble
```

Comment:
We currently run this on 5.9 and 5.10, what coverage do we want moving forward?

Reply:
I just realised that as well. I am currently extracting the matrix based build into a separate workflow so we can easily add more matrix based jobs to it.